tests/Form/Type/AdminTypeTest.php

```diff
@@ -189,8 +189,64 @@ public function testArrayCollection(): void
 
     public function testArrayCollectionNotFound(): void
     {
-        $parentSubject = new \stdClass();
-        $parentSubject->foo = new ArrayCollection([]);
+        $parentSubject = new class() {
+            public $foo = [];
+        };
+
+        $parentAdmin = $this->createMock(AdminInterface::class);
+        $parentAdmin->expects($this->once())->method('getSubject')->willReturn($parentSubject);
+        $parentAdmin->expects($this->once())->method('hasSubject')->willReturn(true);
+        $parentField = $this->createMock(FieldDescriptionInterface::class);
+        $parentField->expects($this->once())->method('setAssociationAdmin')->with($this->isInstanceOf(AdminInterface::class));
+        $parentField->expects($this->once())->method('getAdmin')->willReturn($parentAdmin);
+        $parentField->expects($this->once())->method('getParentAssociationMappings')->willReturn([]);
+        $parentField->expects($this->once())->method('getAssociationMapping')->willReturn(['fieldName' => 'foo', 'mappedBy' => 'bar']);
+
+        $modelManager = $this->createStub(ModelManagerInterface::class);
+
+        $newInstance = new class() {
+            public function setBar()
+            {
+            }
+        };
+
+        $admin = $this->createMock(AbstractAdmin::class);
+        $admin->expects($this->atLeastOnce())->method('hasParentFieldDescription')->willReturn(true);
```

Comment:
I think we can use `exactly()` instead of `atLeastOnce()` here, since this test is not using variable arguments.

Reply:
Done